modules/imgcodecs/perf/perf_png.cpp

```diff
@@ -58,6 +61,30 @@ PERF_TEST(PNG, encode)
     SANITY_CHECK_NOTHING();
 }
 
+PERF_TEST_P(PNG, params,
+    testing::Combine(
+        testing::Values(IMWRITE_PNG_STRATEGY_DEFAULT, IMWRITE_PNG_STRATEGY_FILTERED, IMWRITE_PNG_STRATEGY_HUFFMAN_ONLY, IMWRITE_PNG_STRATEGY_RLE, IMWRITE_PNG_STRATEGY_FIXED),
+        testing::Values(IMWRITE_PNG_NO_FILTERS, IMWRITE_PNG_FILTER_NONE, IMWRITE_PNG_FILTER_SUB, IMWRITE_PNG_FILTER_UP, IMWRITE_PNG_FILTER_AVG, IMWRITE_PNG_FILTER_PAETH, IMWRITE_PNG_FAST_FILTERS, IMWRITE_PNG_ALL_FILTERS),
+        testing::Values(1, 4, 9)))
```

Comment:
`testing::Values(1, 9)` maybe better. Edge values give enough information about output file sizes and compression time